app.js

```diff
@@ -29,6 +32,8 @@ app.use((req, res, next) => {
 app.use("/api/users", usersRoutes.router);
 app.use("/api/admin", adminRoutes.router);
 app.use("/api/discussion", discussionRoutes.router);
+app.use("/api/events-admin", eventsRoutes.router);
+app.use("/api/events-user", eventsURoutes.router);
```

Comment:
You dont need this route.. all your user related event logic should be on the user route that is already existing

Reply:
okay noted